### PR TITLE
Cart page added

### PR DESCRIPTION
**Pull request by ishapatell1**

### CODE REVIEW

**Review by Richa-rani02 — APPROVED** [review]

**Review by spraveenofficial — APPROVED** [review]
Good to merge.

File: Cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    
+    </div>
+    <!--Main Section-->
+    <main class = "cart-container">
+        <div class = "cart-item">
+            <!--Card 1-->
+            <div class="component-example">
+                <div class="card card-ecommerce">
+                   <div class="card-badge">Limited stock </div>
+                  <img class="card-img" src="/images/AirPods/airpods-1.png" />
```
Comment (mnath30) [inline Cart/cart.html:34]: It is good to add the 'alt' to img tags

**Review by mnath30 — APPROVED** [review]
Code looks good. Just add the alt to img tags

File: Cart/cart.html (hunk 1 above)
Comment (ishapatell1) [reply Cart/cart.html:34]: yes , i will keep that in mind

**Pull request merged**